hack/run-in-container.sh

```diff
@@ -37,13 +30,60 @@ DOCKER_GRAPH_VOLUME=${DOCKER_GRAPH_VOLUME:-tidb-operator-docker-graph}
 DOCKER_GO_VOLUME=${DOCKER_GO_VOLUME:-tidb-operator-go}
 NAME=${NAME:-tidb-operator-dev}
 
+function usage() {
+    cat <<'EOF'
+This script is entrypoint to start a isolated container environment for development.
+
+Usage: hack/run-in-container.sh [-h] [command]
+
+    -h      show this message and exit
+
+Environments:
+
+    CLEANUP             if passed, clean up local caches
+    DOCKER_LIB_VOLUME   the name of docker lib volume, defaults: tidb-operator-docker-lib
+    DOCKER_GRAPH_VOLUME the name of docker lib volume, defaults: tidb-operator-docker-graph
+    DOCKER_GO_VOLUME    the name of go cache volume, defaults: tidb-operator-go
+    NAME                the name of container
+
+Examples:
+
+0) view help
+
+    ./hack/run-in-container.sh -h
+
+1) start an interactive shell
+
+    ./hack/run-in-container.sh
+
+    You can start more than one terminals and run `./hack/run-in-container.sh` to
+    enter into the same container for debugging.
+
+2) run command and exit
+
+    ./hack/run-in-container.sh make test
+    ./hack/run-in-container.sh ./hack/e2e.sh -- --ginkgo.focus='aggregated'
+
+3) clean local caches
```

Comment:
Is it better to make it more clear what caches will be cleared?

Reply:
done in 8f84be249f96aa8e8ae53538b26c7ea1e11adce6